source/accessibility.html.md

```diff
@@ -0,0 +1,87 @@
+- Some headings do not fit with the content that follows.
+- Some links do not clearly explain where they lead, or that they lead to external sites.
+- Some pages skip heading levels.
+- Some images do not have good alternative text, or have no alternative text.
+- Some content uses emojis without alternative text.
+- There are some links to documents in PDF format that are not accessible.
+
+## Feedback and contact information
+
+If you need any part of this service in a different format like large print, audio recording or braille, [contact us using the GOV.UK contact form](https://www.gov.uk/contact/govuk).
+
+We’ll consider your request and get back to you within 5 working days.
+
+## Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [contact us using the GOV.UK contact form](https://www.gov.uk/contact/govuk).
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018
+(the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [contact the Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
+
+## Technical information about this website’s accessibility
+
+GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+### Compliance status
+
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to the non-compliances listed below.
+
+### Non-accessible content
+
+The content listed below is non-accessible for the following reasons.
+
+#### Non-compliance with the accessibility regulations
+
+- Some links are broken, which fails WCAG 2.1 success criterion 2.1.1 Keyboard.
+- Some tables are structured in a way that fails WCAG 2.1 success criteria 1.3.1 Info and Relationships and 1.3.2 Meaningful Sequence.
+- Some content uses emojis without alternative text, which fails WCAG 2.1 success criteria 1.1.1 Non-text Content.
+- Some headings do not fit with the content that follows, which fails WCAG 2.1 success criterion 2.4.6 Headings and Labels.
+- Some links do not clearly explain where they lead, or that they lead to external sites. This fails WCAG 2.1 success criteria 2.4.4 Link Purpose (In Context) and 2.4.4 Link Purpose (In Context).
+- Some pages skip heading levels, which fails WCAG 2.1 success criteria 1.3.1 Info and Relationships, 2.4.1 Bypass Blocks and 2.4.6 Headings and Labels.
+- Some images do not have good alternative text or are missing alternative text, which fails WCAG 2.1 success criterion 1.1.1 Non-text Content.
```

Comment:
This list duplicates the one further above - I wonder if it's worth considering keeping to one list to ensure that we don't have to update in multiple places?

Reply:
I'll ask the tech writers and see what they think.

Reply:
Hi Chris - good call and we'd definitely normally avoid this kind of duplication! However we need to follow the structure of the [accessibility statement template](https://www.gov.uk/government/publications/sample-accessibility-statement/), which means we cover the list twice - once without WCAG criteria and once with. Hope that's ok.

Reply:
No problem, just thought I'd check - thanks for the response!